### PR TITLE
Plot non-numerical mnemonic dataset

### PR DESCRIPTION
**Pull request by mfixstsci**
Many of the mnemonic datasets used by the EDB contain `date` and non-numerical `euvalues`.

Current approach:
* If unique possible values of `euvalues` is `> 1 & <=4` then we will plot the data.
* If unique possible values of `euvalues` is `> 4` then we will display a table
* If values are numeric, just generate the expected figure.

### CODE REVIEW

**Comment by mfixstsci** [comment]
Ready for review, I copied the code that "Explore the EDB mnemonic inventory" uses to generate the table. I am sure there is a way to synergize the code there and in the method I wrote to generate a table for mnemonic data with too many unique text values.

Examples to Test:
------------------
* Defaults will return expected plot
* Test parameters that only has 3 or 4 unique values (`SA_ZSLEWST`, `2021-09-01 00:00:00.000`, `2021-09-03 00:01:00.000`)
* Test parameters for text that has too many unique strings to plot (`ICTM_EVENT_MSG`, `2021-09-01 00:00:00.000`, `2021-10-01 00:01:00.000`)

Try it out locally and tell me what you think!

File: jwql/edb/engineering_database.py [hunk 1]
```diff
@@ -142,6 +145,64 @@ def bokeh_plot(self, show_plot=False):
...
+            script, div = components(p1)
+
+            return [div, script]
+
+    def get_table_data(self):
+        """Get data needed to make interactivate table in template.
+        """
+
+        import tempfile
+        import copy
```
Comment (bourque) [inline jwql/edb/engineering_database.py:191]: Could these be moved up to the top of the module with the other imports?

File: jwql/edb/engineering_database.py [hunk 2]
```diff
@@ -45,7 +45,11 @@
 
 from collections import OrderedDict
 from datetime import datetime
+from enum import unique
+# from msilib.schema import tables
 import os
+from select import POLLOUT
+from tokenize import PlainToken
```
Comment (bourque) [inline jwql/edb/engineering_database.py:52]: Could these be removed?  I don't see them being used in the code.

File: jwql/edb/engineering_database.py [hunk 3]
```diff
@@ -45,7 +45,11 @@
 
 from collections import OrderedDict
 from datetime import datetime
+from enum import unique
```
Comment (bourque) [inline jwql/edb/engineering_database.py:48]: I see `numpy.unique` being used in the code, but I don't see `enum.unique` being used.  Could this be removed?

**Review by bourque — CHANGES_REQUESTED** [review]
@mfixstsci Thanks for these changes, they look and work great when I test them out!  I just have a few questions in the review.

**Review by bourque — APPROVED** [review]
Thanks for the updates @mfixstsci, this looks great!

**Pull request merged**